channels_jsonrpc/jsonrpcconsumer.py

```diff
@@ -253,12 +267,12 @@ def __handle(self, content, message):
             else:
                 if isinstance(data, dict):
 
-                    if data.get('method') is not None and data.get('params') is not None and data.get('id') is None:
```

Comment:
@soulhunter1987 
Why did you remove the check for 'params' and 'method'? 
That is what make the tests fail.

If sending {"value": "my_value"} which should be received as a wrongly formatted  JSON-RPC frame, it handles it as a notification,....